assets/css/components/_alert.scss

```diff
@@ -0,0 +1,5 @@
+.alert:empty {
+  @if empty {
```

Comment:
what does this do? 🤔 💭

Reply:
@rosle the `:empty` after `.alert` seem redundant, I removed it.
This for checking the div `.alert` as if it's [empty](https://developer.mozilla.org/en-US/docs/Web/CSS/:empty), then the alert div won't be shown.

Reply:
Why not using the CSS selector `:empty` directly?

Reply:
Because we are using SASS, so I would like to take advantage of its benefits. Using the `@if` to make the code more readable.